environment.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
@kir0ul what are these lines for? I search conda docs and other examples and I can't find mention or other examples of these.

Reply:
Ah I guess it's a document separator in the yaml spec. But this is all one document. I would ask to remove all these, and then this is ready to merge!

Reply:
Ah yes this was suggested by [yamllint](https://github.com/adrienverge/yamllint), it's the [optional document start marker in YAML](https://www.yaml.info/learn/document.html), I'll remove them.